src/compiler/commandLineParser.ts

```diff
@@ -1577,7 +1577,7 @@ namespace ts {
                 errors.push(createCompilerDiagnostic(Diagnostics.Compiler_option_0_requires_a_value_of_type_1, "extends", "string"));
             }
             else {
-                const newBase = configFileName ? getDirectoryPath(toPath(configFileName, basePath, getCanonicalFileName)) : basePath;
+                const newBase = configFileName ? getDirectoryPath(toPath(configFileName, basePath, identity)) : basePath;

```

Comment:
Might want to create a local `toPath` function delegating to `ts.toPath` that explains why we use `identity` here.